PacketDelivery-VC-ReferenceExample/data-consumer/keycloak/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: keycloak
+description: A Helm chart for Kubernetes to deploy Keycloak
+type: application
+version: 1.0.0
+appVersion: "17.0.1"
```

Comment:
Isnt it rather something like 20.x? The plugin is not compatible with <18;) And yes, I know this is most probably a mistake copied from me...